### ISSUE TITLE
Featured banner missing focus state

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#84]
The featured banner on the MDN Web Docs landing page has a CTA button, but it is missing a focus state. Also, ensure that the button has a clear text description for users using assistive technologies.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563468
